Jakefile.js

```diff
@@ -57,6 +57,7 @@ function measure(marker) {
 }
 
 var compilerSources = [
+    "dataStructures.ts",

```

Comment:
I'd be tempted to call this `collections.ts`, since we already have a `types.ts` and this is more specific.